k8s/kube-base/job-xdmod-openshift-prod.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: xdmod-openshift-prod-job
+spec:
+  schedule: "0 1 */1 * *"
+  jobTemplate:
+    spec:
+      concurrencyPolicy: "Forbid"
```

Comment:
shouldn't 'concurrencyPolicy: "Forbid"' be at the same level as 'schedule: "0 1 ..."'?

Reply:
Oh, whoops - you're right. Fixed!